src/__tests__/store/ui/reducer.test.ts

```diff
@@ -0,0 +1,141 @@
+import { UiActionTypes, IUiState } from "../../../store/ui/types";
+import { uiReducer } from "../../../store/ui/reducer";
+
+const initialState: IUiState = {
+    loading: false,
+    progress: 0,
+    isDropdownOpen: false,
+    isKebabDropdownOpen: false,
+    isSidebarOpen: true,
+    sidebarActiveItem: "dashboard",
+    sidebarActiveGroup: "feeds_grp"
+};
+
+describe('ui reducer', () => {
+    it('should return the initial state', () => {
+        expect(uiReducer(undefined, {
+            type: null
+        })).toEqual(
+            initialState
+        )
+    })
+
+    it('should fetch request', () => {
```

Comment:
the test state is not needed here. As you can see by the reducer there are no inputs to the fetchrequest action. We are just letting the reducer know the fetch request has started and to toggle the loading state. No payload required.